src/dotnet/Program.cs

```diff
@@ -178,6 +182,9 @@ private static void ConfigureDotNetForFirstTimeUse()
                             environmentProvider);
 
                         dotnetConfigurer.Configure();
+
+                        if (!File.Exists(Telemetry.TelemetrySentinel))
```

Comment:
With a machine wide installation of dotnet, shared between two users, the first time invoking that machine wide installation for both users will trigger the first time experience, because the cache is user-local by default. 

However, the TelemetrySentinel is dotnet-local so in this case would be machine wide. If one user has gone through first time and seen the first time use message, the TelemetrySentinel will be written. If the second user were not to go through the first time experience, the TelemetrySentinel would still be written and Telemetry would be enabled.

Is that an intended scenario?